package.json

```diff
@@ -40,6 +43,10 @@
     "sinon": "^2.1.0",
     "sinon-chai": "^2.9.0",
     "style-loader": "^0.14.1",
-    "webpack": "^2.2.1"
+    "webpack": "^2.2.1",
+    "webpack-node-externals": "^1.5.4"
+  },
+  "dependencies": {
+    "hpq": "^1.0.1"
```

Comment:
I really like this library, great job 👍 

Minor but do you think we should move it inside the project or something?